docker-compose.yml

```
@@ -46,6 +46,7 @@ services:
       - hapi.fhir.delete_enabled=true
       - hapi.fhir.expunge_enabled=true
       - hapi.fhir.enforce_referential_integrity_on_delete=false
+      - hapi.fhir.enforce_referential_integrity_on_write=false 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)